x11-terms/guake/guake-3.9.0.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
Doesn't it works against python 3.11?

Reply:
> Doesn't it works against python 3.11?

Untested. I am on 3.10 currently. 
Reluctant to switch. Please don't ask me? :-)

Reply:
Python 3.11 will be the stable version in no time: you'd better test it

Reply:
I have no problem against it. Mine was a suggestion just to drop the future PR but you may want to ask to a dev too.

Reply:
The package is currently utterly abandoned and unmaintained. I am using it on my laptop, but my laptop was also abandoned and broken for a while. I recently repaired it enough for it to start working, and now I'm trying to bring it back into some kind of decent condition. New ssd, new fan, new battery... new version of guake. If both survive the process, they will live. :-) The laptop still needs a new wifi adapter and a new rechargeable RTC battery. And guake needs support for new Python. The surgery is still in progress. :-)

Reply:
> but you may want to ask to a dev too.

Did you mean that I could join the developers and become a developer myself?

Reply:
> > but you may want to ask to a dev too.
> 
> Did you mean that I could join the developers and become a developer myself?

No. I mean you have to ask to a dev if it is ok to add python3.11 in a 2nd PR

Reply:
Ah, okay. I got confused for a second there. :-)